Sources/Testing/Issues/Issue.swift

```diff
@@ -215,6 +215,7 @@ extension Issue {
     public var sourceContext: SourceContext
 
     /// Whether or not this issue is known to occur.
+    @_spi(ForToolsIntegrationOnly)
```

Comment:
This change was intended to be applied to the property in `Issue`, not `Issue.Snapshot`, wasn't it?

Reply:
🤦